lib/generators/geoblacklight/templates/settings.yml

```diff
@@ -21,24 +21,31 @@ OVERLAP_RATIO_BOOST: '2'
 
 # Solr field mappings
 FIELDS:
+  :CREATOR: 'dc_creator_sm'
+  :DESCRIPTION: 'dc_description_s'
   :FILE_FORMAT: 'dc_format_s'
+  :GEOM_TYPE: 'layer_geom_type_s'
   :GEOMETRY: 'solr_geom'
-  :WXS_IDENTIFIER: 'layer_id_s'
-  :RIGHTS: 'dc_rights_s'
+  :IDENTIFIER: 'dc_identifier_s'
+  :ISSUED: 'dct_issued_s'
+  :LANGUAGE: 'dc_language_s'
+  :LAYER_ID: 'layer_id_s'
```

Comment:
Is this redundant w/ `WXS_IDENTIFIER`?

Reply:
Yeah, I like that idea for adding defaults. My LAYER_ID is a duplication of WXS_IDENTIFIER. The WXS_ prefix threw me. I wasn't sure what that intended (must be mappy).

Are you thinking something like...

```Ruby
module Geoblacklight
  class Engine < ::Rails::Engine
    Blacklight::Configuration.default_values[:view].split.partials = ['index']
    Blacklight::Configuration.default_values[:view].delete_field('list')
    
    # Settings.FIELDS Default Values
    Settings.FIELDS.IDENTIFIER = 'dc_identifier_s' unless Settings.FIELDS.IDENTIFIER.present?
    # ... A bunch more defaults here ...

    # GeoblacklightHelper is needed by all helpers, so we inject it
    # into action view base here.
    initializer 'geoblacklight.helpers' do
      ActionView::Base.send :include, GeoblacklightHelper
      ActionView::Base.send :include, CartoHelper
    end

    config.to_prepare do
      Geoblacklight.inject!
    end
  end
end
```

Can you access Settings in engine.rb?

Reply:
Yep something like that. I _think_ we could even:

```
    initializer 'config.initialize' do #referencing config gem
    #check if it exists and maybe set a default?
    # or even
    # Config.load_files('new_values_only.yml')
    end
```

Reply:
Okok. Will attempt on Monday.